themes/default/content/blog/pulumi-release-notes-70/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Pulumi Release Notes: Display richer diffs, clear pending operations, and much more!"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-04-19T07:26:49-07:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: The latest Pulumi updates also include our providers updates, install Pulumi using winget, stack unselect command, GitHub release private plugins, and more.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+The team has been busy releasing new features and improvements in the last month. Read on to learn about what's new in this release!
+
+- Cloud Providers and Packages
+  - [Pulumi AWS Provider v5.0.0](#pulumi-aws-provider-v5.0.0)
```

Comment:
Only note; not a blocker: I'd recommend moving the `<!--more-->` above this list on line 36 because if someone attempts to select the links here from the main blog page, it won't lead anywhere.

Reply:
Ah, sorry @meagancojocar -- this is my fault. I think I was the one who mentioned these links didn't have to be relrefs, as they anchored into the same page -- but I forgot that we sometimes surface these links outside of the page. Using a relref would work for both situations. Apologies for leading you astray!